cafebabel/tests/test_translations.py

```diff
@@ -326,7 +326,7 @@ def test_translation_published_should_have_translator(
              f'article title') in response)
     assert (f'<a href="/fr/profile/{translator.id}/" class=translator-link>'
             in response)
-    assert f'{translator}' in response
+    assert translator in response
```

Comment:
Tests are failing because of this, maybe `str(translator) in response` would be enough.

Reply:
You sure?!

Reply:
Yes but the pushed had been rejected after master update :/